### PR TITLE
I13 add expense screen

### PR DESCRIPTION
**Pull request by HKanwal**
## Description
Created component for dropdown input. Added functionality for filter and for selecting from dropdown for merchant and category. Made details input box responsive to content. Not done sending the mutation request or the calendar date picker or form validation.

## Related Issue
- [x] This pull request relates to at least one particular issue

  
  - [Add Expense](https://github.com/ps-toronto-team-4/.github/issues/13)

## Checks
- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information

To test, simply launch the app, login, and click the "+" button on the expense details screen in the bottom right corner.

### CODE REVIEW

**Comment by HKanwal** [comment]
Don't delete the branch.

File: components/AdaptiveTextInput.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+export default function AdaptiveTextInput(props: AdaptiveTextInputProps) {
+    const { style, value, ...otherProps } = props;
+    const [ fontLoaded ] = useFonts({B612Mono_700Bold});
+
+    if (!fontLoaded) {
+        return <View></View>
+    }
+
+    return (
+        <>
```
Comment (1mozolacal) [inline components/AdaptiveTextInput.tsx:29]: The <> is not necessary as the <TextInput> can be the root component.

**Review by 1mozolacal — APPROVED** [review]

File: components/AdaptiveTextInput.tsx (hunk 1 above)
Comment (HKanwal) [reply components/AdaptiveTextInput.tsx]: Fixed

**Comment by 1mozolacal** [comment]
Does adding date work?

**Comment by HKanwal** [comment]
No not yet.

**Pull request merged**